cmd/geth/accountcmd.go

```diff
@@ -222,6 +223,74 @@ func accountList(ctx *cli.Context) error {
 	return nil
 }
 
+// tries unlocking the specified account a few times.
+func unlockAccount(ks *keystore.KeyStore, address string, i int, passwords []string) (accounts.Account, string) {
+	account, err := utils.MakeAddress(ks, address)
+	if err != nil {
+		utils.Fatalf("Could not list accounts: %v", err)
+	}
+
+	for trials := 0; trials < 3; trials++ {
+		prompt := fmt.Sprintf("Unlocking account %s | Attempt %d/%d", address, trials+1, 3)
+		password := utils.GetPassPhraseWithList(prompt, false, i, passwords)
+
+		err = ks.Unlock(account, password)
+		if err == nil {
+			log.Info("Unlocked account", "address", account.Address.Hex())
+			return account, password
+		}
+
+		if err, ok := err.(*keystore.AmbiguousAddrError); ok {
```

Comment:
Maybe we should use the following here as the type assertion on errors could fail on wrapped errors 
```
var err *keystore.AmbiguousAddrError
if errors.As(err, &err)
```

Reply:
After removing unused function that code is gone.